routes/web.php

```diff
@@ -154,20 +155,20 @@
 /**
  * RECORDS
  */
-Route::middleware('authWeb:web')->get('/verified/records', function () {
+Route::middleware('authWeb:web')->get('/records', function () {
```

Comment:
I think we require `/verified` potentially unless this did work for you when changing it